### DISCUSSION TITLE
Running tailwindcss is required for some tests but breaks some others.

### DISCUSSION THREAD

**Discussion by rafa-dyad** [keathley/wallaby#662] [Q&A]
## Issue

Running tailwindcss is required for some tests but breaks some others.

## Details

Elixir: latest bitwalker/alpine-elixir-phoenix
Erlang:  latest bitwalker/alpine-elixir-phoenix
OS: Alpine
Webdriver , chromedriver
Browser chromium
Local or CI:  CI, codefresh running a bitwalker elixir image

Essentially, the issue is that adding tailwindcss makes some test fail, but it seems to be essential for some others. This seems unrelated to the test itself. None of them particularly care about css styles, or tailwind in particular, and they fail with unrelated errors.

Test 1, failing when tailwind is added with a chromium invalid session id error.

Test 2, failing when tailwind is not added with Uncaught DOMException: Failed 
to execute 'replaceState' on 'History'

We have isolated the problem: running test 1, then running tailwindcss, then running test 2 is the only way to make both pass. 

Any ideas about the relationship between those errors and tailwindcss? This is the line that makes or breaks a test:
"NODE_ENV=production tailwindcss --postcss --minify --input=css/app.css --output=../priv/static/css/app.css"

Thank you

**Comment by mhanberg** [comment]
If you have css that changes the visibility of elements, you'll need to make sure that it's generated and up to date before your test runs.

**Reply by rafa-dyad** [reply]
Thanks @mhanberg . I don't think that's the issue in this particular example.
To be honest I'm more concerned about the invalid session id errors.  Have you encountered those?

**Reply by mhanberg** [reply]
that usually means that chromedriver/chrome crashed or failed to start. can you confirm the versions of both?
